partner_export_wamas/models/res_partner.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _wamas_export(self, specific_dict=None, telegram=False):
+        """
+        Export the partner data as WAMAS format
+
+        :return: partner data as WAMAS format
+        :rtype: bytes
+        """
+        self.ensure_one()
+        if not telegram:
+            raise ValueError(_("Need telegram for exporting"))
+        # If having a specific dict for the partner, we use it
+        if specific_dict and isinstance(specific_dict, dict):
+            dict_partner = specific_dict
+        else:
+            dict_partner = self.read()[0]
+        base_wamas_ubl = self.env["base.wamas.ubl"]
+        return base_wamas_ubl.dict2wamas(dict_partner, telegram)
```

Comment:
To answer my point above regarding depending on _ubl module: IMO here we should look at the right helper model via configuration/parameter and use always the same method to get the data.
Something like:
```suggestion
    def _wamas_export(self, specific_dict=None, telegram=False, export_format=None):
        """
        Export the partner data as WAMAS format

        :return: partner data as WAMAS format
        :rtype: bytes
        """
        self.ensure_one()
        if not telegram:
            raise ValueError(_("Need telegram for exporting"))
        # If having a specific dict for the partner, we use it
        if specific_dict and isinstance(specific_dict, dict):
            dict_partner = specific_dict
        else:
            dict_partner = self.read()[0]

        export_format = export_format or self.export_format  # if we want to make it configurable by partner
        handler = self._wamas_export_get_handler(export_format)
        return handler.dict2wamas()

    def _wamas_export_get_handler(self, export_format):
        try:
            return self.env[f"base.wamas.{export_format}"]
        except KeyError:
            raise UserError(_("Not supported format %(fmt)s", fmt=export_format))
```
can you please add a TODO in the method at list?